app/views/forms/view/view-contact.directive.html

```diff
@@ -78,15 +78,15 @@ <h4>
 			<address class="km-value">
 				<div class="km-pre">{{document.address | lstring:locale}}</div>
 				{{document.city | lstring:locale}}
-				<span ng-show="document.city">, </span>
+				<span ng-show="document.city">,</span>
 				{{document.state | lstring:locale}}
-				<br ng-show="document.city || document.state" /> {{document.postalCode | lstring:locale}}
-				<span ng-show="document.postalCode">, </span>
+				<br ng-show="document.city || document.state" />{{document.postalCode | lstring:locale}}
```

Comment:
isn't better to use ng-bind from translation?
```suggestion
				<br ng-show="document.city || document.state" />
				<span ng-bing="document.postalCode | lstring:locale"></span>
```